cypress/integration/views/commoncontrols.view.ts

```diff
@@ -13,5 +13,6 @@ export const itemsPerPageMenuOptions = "ul[aria-labelledby='pagination-options-m
 export const expandRow = "button[aria-label=Details]";
 export const successAlertMessage = "div[aria-label='Success Alert']";
 export const duplicateNameWarning = "div[aria-label='Danger Alert']";
+
 export const nameHelper = "div[id=name-helper]";
```

Comment:
remove this empty line

Reply:
Yes, this reminds me that, I kept it to discuss about the nameHelper and descriptionHelper. These two should not be placed in commons, as they are specific to stakeholdergroup page view only.